usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
@ltrilety  @dahorak I will leave common tests in ```usmqe_tests/api/tendrl```. Once we have API tests for login we can remove skyring tests from repo and have directory tree more flat. What do you think about that?